conv_model_upd.cpp

```diff
@@ -436,6 +479,13 @@ int conv_benchmark(int argc, char** argv) {
   long _r_step = 1;
   long _s_step = 1;
 
+  printf("Test parameters: N H W C K R S stride_h stride_w pad_h pad_w bc bk logical_padding: %d %d %d %d %d %d %d %d %d %d %d %d %d %d \n", N, H, W, C, K, R, S, stride_h, stride_w, pad_h, pad_w, bc, bk, logical_padding);
```

Comment:
Some more verbose output, I'd suggest keeping it since there is anyway tons of output and this allows to see if the parameters propagated correctly through the heuristics.